src/vector/platform/ElectronPlatform.js

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
 import VectorBasePlatform from './VectorBasePlatform';
 import dis from 'matrix-react-sdk/lib/dispatcher';
+import _t from 'matrix-react-sdk/lib/languageHandler';
```

Comment:
Yep, typically I had only tested this on `WebPlatform` which was the only one to import correctly. Thanks.